doc/source/whatsnew/v0.25.2.rst

```diff
@@ -86,7 +86,7 @@ Groupby/resample/rolling
 Reshaping
 ^^^^^^^^^
 
--
+- Fix to ensure all int dtypes can be used in :meth:`pandas.core.reshape.merge_asof` when using a tolerance value (:issue:`28870`).
```

Comment:
`` :func:`merge_asof` `` and maybe clarify that any non-int64 integer previously raised an erroneous `MergeError`